frontend/src/composables/WorkSpace/useDomainData.js

```diff
@@ -148,10 +141,8 @@ export default function useDomainData() {
 	getTagsTree();
 
 	const OnDelete = async (id) => {
-		console.log("deleted ID:", id);
 		await API.delete(`/domain/${choosenNodeID.value}/tag/${id}`)
 			.then(() => refreshDomainTag(choosenNodeID.value))
-			.catch((error) => console.log(error));
```

Comment:
Could be good to add error handling later on! What is happening for now if the API call is in error? Should we write a US to handle this error?

Reply:
Yes all the API calls will be handled globally on the axios middleware